src/ui/digital-exchange/RatingFilterItem.js

```diff
@@ -0,0 +1,51 @@
+import { range } from 'lodash';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { formattedText } from '@entando/utils';
+import StarIcon from 'ui/digital-exchange/common/StarIcon';
+
+const RatingFilterItem = ({
+  selected,
+  rating,
+  maxRating,
+  onSelect,
+}) => {
+  const selectedClass = selected ? ' RatingFilterItem--selected' : '';
+  const className = `RatingFilterItem${selectedClass}`;
+  const enterKeyCode = 13;
+  const itemLabel = formattedText('digitalExchange.sidebar.ratingFilter.itemLabel');
+  return (
+    <div
+      role="button"
+      tabIndex="0"
+      className={className}
+      onClick={() => onSelect(rating)}
+      onKeyDown={e => e.keyCode === enterKeyCode && onSelect(rating)}
```

Comment:
`keyCode` is deprecated: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode

Reply:
fixed in commit 94e21ab0189f339d67a26b342b6df9905ab79739